src/ansys_sphinx_theme/theme/ansys_sphinx_theme/_templates/autoapi/python/class.rst

```diff
@@ -14,7 +14,38 @@
             - {{ obj.summary }}
           {% endfor %}
 {%- endmacro %}
-{# --------------------------- End macros definition ----------------------- #}
+
+{# ------------------------ End macros definition for tab ------------------- #}
+
+{# ----------------- Start macros definition for autosummary -----------------#}
+
+{% macro add_auto_summary_attribute(heading, id, types) -%}
+
+{{ heading }}
+{{ "-" * heading | length }}
+
+.. autoapisummary::
+
+         {% for type in types %}
+    {{ type.id }}
+        {% endfor %}
+
+{%- endmacro %}
+{# ------------------ End macros definition for autosummary --------------- #}
+
+{# ----------------- Start macros definition for headers -----------------#}
+{% macro render_header_with_attributes(types, title) -%}
```

Comment:
Just for consistency, could we use `heading` instead of title here? Also, could we follow the same parameter order as in previous macro?

Reply:
yes, i will add that

Reply:
renamed heading to title [e333f4c](https://github.com/ansys/ansys-sphinx-theme/pull/556/commits/e333f4cf20319e4d800e7a93f657076ed5898d87)

Reply:
Still need to fix the order.

Reply:
right, i will do that.